open-metadata-implementation/access-services/glossary-view/glossary-view-spring/pom.xml

```diff
@@ -54,9 +54,9 @@
         </dependency>
 
         <dependency>
-            <groupId>javax.validation</groupId>
-            <artifactId>validation-api</artifactId>
-            <version>2.0.1.Final</version>
+            <groupId>org.springframework.boot</groupId>
```

Comment:
As this is a new dependency you'll now need to add to the top level pom an entry IN THE DEPENDENCYMANAGEMENT section  like
```
          <dependency>
                <groupId>org.springframework.boot</groupId>
                <artifactId>spring-boot-starter-validation</artifactId>
                <scope>compile</scope>
                <version>${spring-boot.version}</version>
            </dependency>
```
Then in this pom do not include the version tag

Reply:
Done. Thank you for the explanation